drivers/hmis/app/graphql/types/application/user.rb

```diff
@@ -0,0 +1,31 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class Application::User < Types::BaseObject
+    description 'User account for a user of the system'
+    graphql_name 'ApplicationUser'
+    field :id, ID, null: false
+    field :name, String, null: false
+    field :recent_items, [Types::HmisSchema::OmnisearchResult], null: false
+    field :date_updated, GraphQL::Types::ISO8601DateTime, null: false
+    field :date_created, GraphQL::Types::ISO8601DateTime, null: false
+    field :date_deleted, GraphQL::Types::ISO8601DateTime, null: true
+
+    def name
+      [object.user_first_name, object.user_last_name].compact.join(' ')
```

Comment:
Is the `object` one of `Hmis::User`?
If so, it has `def name` that'll return the name.